drivers/mlx90393/include/mlx90393_params.h

```diff
@@ -84,20 +88,25 @@ extern "C" {
 #define MLX90393_PARAM_DIG_FILT         (MLX90393_DIG_FILT_1)
 #endif
 
+#if IS_USED(MODULE_MLX90393_WOC) || DOXYGEN
 #ifndef MLX90393_PARAM_THRESHOLD
 /**
  * @brief   Default thresholds for Wake-up On Change mode
  */
 #define MLX90393_PARAM_THRESHOLD                    \
+.threshold =                                        \
 {                                                   \
     .xy             = 0xFFFF,                       \
     .z              = 1000,                         \
     .temp           = 0xFFFF                        \
-}
+},
+#endif
+#else
+#define MLX90393_PARAM_THRESHOLD
```

Comment:
Das Gleiche wie bei der Definition des Interrupt-Pin, sowas kann man nicht überschreiben, zumindest nicht an der Make-Kommandozeile. Daher wie folgt:

```suggestion
#ifndef MLX90393_PARAM_THRESHOLD_XY
#define MLX90393_PARAM_THRESHOLD_XY     0xFFFF
#endif

#ifndef MLX90393_PARAM_THRESHOLD_Z
#define MLX90393_PARAM_THRESHOLD_Z      1000
#endif

#ifndef MLX90393_PARAM_THRESHOLD_TEMP
#define MLX90393_PARAM_THRESHOLD_TEMP   0xFFFF 
#endif

#if IS_USED(MODULE_MLX90393_WOC) || DOXYGEN
#ifndef MLX90393_PARAM_THRESHOLD
/**
 * @brief   Default thresholds for Wake-up On Change mode
 */
#define MLX90393_PARAM_THRESHOLD                        \
.threshold = {                                          \
                 .xy   = MLX90393_PARAM_THRESHOLD_XY,   \
                 .z    = MLX90393_PARAM_THRESHOLD_Z,    \
                 .temp = MLX90393_PARAM_THRESHOLD_TEMP, \
             },
#endif
#else
#define MLX90393_PARAM_THRESHOLD
```